controller/soap_web_service.rst

```diff
@@ -50,56 +50,68 @@ In this case, the SOAP service will allow the client to call a method called
         }
     }
 
-Next, you can train Symfony to be able to create an instance of this class.
-Since the class sends an email, it's been designed to accept a ``Swift_Mailer``
-instance. Using the Service Container, you can configure Symfony to construct
-a ``HelloService`` object properly:
+Next, make sure that your new class is registered as a service. If you use
+:doc:`autowiring </service_container/autowiring>` (enabled by default), this is easy:
 
 .. configuration-block::
 
     .. code-block:: yaml
 
         # app/config/services.yml
         services:
-            hello_service:
-                class: Acme\SoapBundle\Services\HelloService
-                arguments: ['@mailer']
+            _defaults:
+                # be sure autowire is enabled
+                autowire: true
+
+            # load services from the Service directory
```

Comment:
I like your's better... at the very least because it has the important word - `Service` earlier.